### ISSUE TITLE
Crash at "could not process block from fork choice service.."

### ISSUE THREAD

**Issue by mtbitcoin** [prysmaticlabs/prysm#4412]
Running commit 3b3f2c7 , receiving the following before a crash

[2020-01-05 12:17:57]  INFO initial-sync: Processing block 383425/394749 - estimated time remaining 1h17m2s blocksPerSecond=2.5 peers=4/5
[2020-01-05 12:17:58]  INFO initial-sync: Processing block 383429/394749 - estimated time remaining 1h15m28s blocksPerSecond=2.5 peers=4/5
[2020-01-05 12:17:58]  INFO initial-sync: Processing block 383437/394749 - estimated time remaining 1h13m56s blocksPerSecond=2.5 peers=4/5
[2020-01-05 12:17:59]  INFO initial-sync: Processing block 383461/394749 - estimated time remaining 1h12m21s blocksPerSecond=2.6 peers=1/5
panic: could not process block from fork choice service: pre state of slot 383461 does not exist

goroutine 211 [running]:
github.com/prysmaticlabs/prysm/beacon-chain/sync/initial-sync.(*Service).Start(0xc001891680)
        beacon-chain/sync/initial-sync/service.go:119 +0xd0d
created by github.com/prysmaticlabs/prysm/shared.(*ServiceRegistry).StartAll
        shared/service_registry.go:44 +0x23e

**Comment by somuchid** [comment]
same situation here. panic on multiple nodes last night. could not resync. always panic or stuck on around block 383xxx. looks like the  problem of 2 days ago resurface.

did a resync from block 1 last night, no improvement at all, below is output:
[2020-01-05 10:56:29]  INFO blockchain: Finished applying state transition attestations=12 deposits=0 slot=384556
[2020-01-05 10:56:29]  INFO initial-sync: Synced up to slot 384556
[2020-01-05 10:56:38]  INFO sync: Requesting parent block currentSlot=394343 parentRoot=e53ae0bf7a13867d55210623eb73a761858a7c7d6329654f862008117253d755
[2020-01-05 10:56:42]  INFO sync: Requesting parent block currentSlot=394343 parentRoot=e53ae0bf7a13867d55210623eb73a761858a7c7d6329654f862008117253d755
[2020-01-05 10:56:46]  INFO sync: Requesting parent block currentSlot=394342 parentRoot=8d335275f3a99f01028d3d75a2d716e8e9af05a5d95698465f617e31731fc2ed

it says "Synced up to slot 384556". that is wrong. happened multiple times, around block #38xxxx.
then it start looking for backwards for parent block, backing up to parent nodes in merkle tree, the block# continues reducing, from 394342, to 394341,  to now 393923, and still counting

last night, one of my windows system crashed twice when the beacon node is doing resync and searching endlessly for parent tree node

I increased the resource allocated to docker on another machine, to 8gb ram, the machine has not crashed. it run whole night, until now I manually stop the beacon node.

this was error message from last friday: panic: could not process block from fork choice service: pre state of slot 383113 does not exist

saw it last night, on different block, around 38xxxx.

**Comment by mtbitcoin** [comment]
I am seeing the same also around block 38x K

**Comment by rauljordan** [comment]
Duplicate of #4156

**Comment by rauljordan** [comment]
No longer seen after #4099 and #4413
